pkg/controller/disk/disk_controller.go

```diff
@@ -74,7 +74,6 @@ type ReconcileDisk struct {
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileDisk) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
```

Comment:
This declaration has to be removed. Else it will cause `declared not used` error.